packages/desktop/package.json

```
@@ -48,7 +48,7 @@
         "webpack-dev-server": "^3.11.2"
     },
     "build": {
-        "productName": "Firefly",
+        "productName": "Firefly Beta",
```

Revert back before merge:
```suggestion
        "productName": "Firefly",
```